front/lib/api/assistant/actions/conversation/list_files.ts

```diff
@@ -53,9 +57,14 @@ export class ConversationListFilesAction extends BaseAction {
       // TODO(spolu): add mention of viz if enabled and other tools.
       `\n`;
     for (const f of this.files) {
-      content +=
-        `<file id="${f.fileId}" name="${_.escape(f.title)}" type="${f.contentType}" ` +
-        `includable="${isConversationIncludableFileContentType(f.contentType)}" queryable="${!!f.snippet}"`;
+      const readyForJIT = !!f.snippet;
+      const includable = isConversationIncludableFileContentType(f.contentType);
+      const queryable =
+        readyForJIT && isConversationQueryableFileContentType(f.contentType);
+      const searchable =
+        readyForJIT && isConversationSearchableFileContentType(f.contentType);
+
+      content += `<file id="${f.fileId}" name="${_.escape(f.title)}" type="${f.contentType}" includable="${includable}" queryable="${queryable}" searchable="${searchable}"`;
```

Comment:
I think you need to add a bit of prompting around that right? Something like
```
// includable: can be retrieved with the `include_conversation_file`  action
// queryable: can be queried with the `query_conversation_tables` action
// searchable: can be searched with the `….` action
```
Right above the lsit?

Reply:
Seems to work fine with GPT, claude and gemini but I can do it :)

Reply:
Now that we put it in the action description, do you think we need the extra prompting in the output of this action ?